docs/source/search-syntax.rst

```diff
@@ -52,7 +52,7 @@ Entity Name Contains Special Characters
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 When a metric, parameter, or tag name contains a special character like hyphen, space, period, and so on,
-enclose the entity name in double quotes.
+enclose the entity name in double quotes. Using backticks also work.
```

Comment:
enclose the entity name in double quotes. Using backticks also work. -> enclose the entity name in double quotes or  backticks.